Source/Editor/DualityEditor/Controls/AutoExpandTreeView.cs

```diff
@@ -0,0 +1,68 @@
+using Aga.Controls.Tree;
+using System;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace Duality.Editor.Controls
+{
+	public class AutoExpandTreeView : TreeViewAdv
+	{
+		public int DragOverExpandDelay
+		{
+			get;
+			set;
+		} = 1000;
+
+		private TreeNodeAdv autoExpandNode = null;
```

Comment:
Code Style: All fields as a group, no empty lines between them except to signify larger groups of different categories of fields (none in this case). Also, fields are the first thing in a class definition, followed by properties, and then methods.